README.md

```diff
@@ -1 +1,82 @@
-Paketo Composer Buildpack
+# PHP Composer Distribution Cloud Native Buildpack
+
+This buildpack provides the [composer](https://getcomposer.org/) dependency by installing the composer binary
+onto the build image `$PATH` which makes it available to subsequent buildpacks.
+This buildpack will not issue commands to `composer` (such as `composer install`).
+
+A usage example can be found in the
+[`samples` repository under the `php/composer` directory](https://github.com/paketo-buildpacks/samples/tree/main/php/composer).
+
+## Detection
+
+Will add these requires/provisions to the build plan if and only if a `composer.json` file is found.
+
+### Requires:
+
+None
+
+### Provides:
+- `composer`
+
+## Build
+
+Will install Composer at a location on the `$PATH` of the build image for subsequence buildpacks to use.
```

Comment:
```suggestion
Will install Composer at a location on the `$PATH` of the build image for subsequent buildpacks to use.
```

Reply:
I'll change it to say "build or launch" image as well. Thanks for the catch.